### PR TITLE
Rename to doctr and add a setup.py

### LINKED ISSUE #1: Better name

### PR DESCRIPTION
**Pull request by asmeurer**
Feel free to push fixes to this branch. Stuff won't work until the repo name is changed.

I am using versioneer, so to do a release, you just tag a commit with the version number for the release. 

Fixes #1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+License: MIT
+
+""",
+    entry_points={'console_scripts': [ 'doctr = doctr.__main__:main']},
+    install_requires=[
+        'requests',
+        'cryptography',
+    ],
+    license="MIT",
+    classifiers=[
```
Comment (asmeurer) [inline setup.py]: Am I missing any?

Comment (gforsyth) [inline setup.py]: Not that I can see.

**Comment by asmeurer** [comment]
I pushed a release to PyPI. What is your PyPI username?

**Comment by gforsyth** [comment]
it's gforsyth

**Pull request merged**